### PR TITLE
chore: reduce number of e2e tests and other fixes

### PR DESCRIPTION
**Pull request by arshadali172**
## Problem

Closes https://github.com/opengovsg/FormSG/issues/302

## Solution

- Run tests in either full or basic environment. Do not run the same test in both environments. This helps shorten test time.
- Add `--skip-js-errors` flag that is causing some encryp mode tests to fail
- Revert package upgrade for testcafe

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -233,7 +233,7 @@
     "stylelint-prettier": "^1.1.2",
     "supertest": "^3.3.0",
     "terser-webpack-plugin": "^1.2.3",
-    "testcafe": "^1.9.1",
+    "testcafe": "^1.8.0",
```
Comment (mantariksh) [inline package.json:236]: hm I was experimenting with this - I think you need `npm install --save-dev testcafe@1.8.6` (note 1.8.6, which was the previous version) in order to propagate the changes to `package-lock.json`

**Review by mantariksh — APPROVED** [review]
lgtm, except I think we need to propagate version `1.8.6` to `package-lock.json`

**Comment by liangyuanruo** [comment]
Do you need to re-enable E2E tests on Travis?

File: package.json (hunk 1 above)
Comment (arshadali172) [reply package.json:236]: roger

**Comment by arshadali172** [comment]
> Do you need to re-enable E2E tests on Travis?

on point. thanks for the catch!

**Pull request merged**